packages/graphql_server/src/test/githubApi.test.ts

```diff
@@ -7,9 +8,34 @@ export async function testGetRepositoryTopics(repoFounder: string, repoName: str
 
 //test function that calls the method and prints out all contributors
 export async function testGetContributorCount(owner: string, repo: string) {
-  console.log(await getContributorCount(owner, repo, ' ')) //here we need to pass an auth token
+  console.log(await getContributorCount(owner, repo, '')) //here we need to pass an auth token
```

Comment:
I know you did not create this test, but please:
-  use` process.env.GITHUB_API_TOKEN` instead of empty string
- log the repo name as well like: `console.log(`${owner}/${name}:`)` or similar